views/Transactions/Transactions.js

```diff
@@ -93,36 +95,17 @@ export function Transactions() {
               <Spinner color='amber.400' />
             </Center>
           ) : transactions.length <= 0 ? (
-            <Center flex={0.9}>
-              <Center flex={1} key='ListEmptyComponent-C2'>
-                <Icon
-                  as={AntDesign}
-                  name='star'
-                  color='yellow.400'
-                  size={{ base: 'lg', sm: 'xl' }}
-                  mb={{ base: '14px', sm: '20px' }}
-                  key='ListEmptyComponent-Icon'
-                />
-                <Heading
-                  key='ListEmptyComponent-H2'
-                  size='md'
-                  fontSize={{ base: '17px', sm: '20px' }}
-                  textAlign='center'
-                  lineHeight='30px'
-                  mb={{ base: '18px', sm: '32px' }}
-                >
-                  To get started, send DOGE to your wallet
-                </Heading>
-                <HStack width='86%' key='ListEmptyComponent-HS2'>
-                  <Button
-                    key='ListEmptyComponent-BB4'
-                    onPress={() => setAddressDetailOpen(true)}
-                  >
-                    Deposit DOGE
-                  </Button>
-                </HStack>
-              </Center>
-            </Center>
+            <VStack pt='48px' alignItems='center'>
+              <Text color='gray.500' pt='24px' pb='32px'>
+                No transactions found
+              </Text>
+              <Text fontSize='16px'>
+                To get started, send DOGE to your wallet
+              </Text>
+              <BigButton mt='24px' onPress={openReceiveModal}>
+                Deposit DOGE
+              </BigButton>
```

Comment:
Looks good. We should also add a button here to open buy.getdoge.com?address=<address> similar to the mobile app.